docs/source/datasets.rst

```diff
@@ -0,0 +1,72 @@
+Datasets
+========================
+To perform an automated systematic review, ASReview requires a dataset of papers.
+The dataset of papers usually results from an online library search for all studies related to the topic to review.
```

Comment:
Perhaps:
"To create such a dataset for a systematic review, typically an online library search is performed for all studies related to a particular topic."
?